### DISCUSSION TITLE
Airflow webserver and worker constantly restart everyday at the same time

### DISCUSSION THREAD

**Discussion by robinisme2** [apache/airflow#43364] [General]
### Official Helm Chart version

1.13.1

### Apache Airflow version

2.8.3

### Kubernetes Version

1.30.4

### Docker Image customizations

No

### What happened

I created an airflow application by helm chart on my k8s cluster, and I found that the worker and webserver will restart everyday at the same time for no reason, there's nothing special in the log, and also not liveness issue. Besides, the helm chart is deployed by argocd, and I found that the argocd sync time is exactly the same as the pod restart time. 

### How to reproduce

It happens everyday at the same time

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by nathadfield** [comment]
@robinisme2 I don't think there's much we can do to help you here currently because this is more than likely a configuration/implementation issue on your side rather than an Airflow bug AFAIK.  If you are able to identify a bug, validate that it still exists in the latest version and then provide a way for anyone to reproduce it then you might get the assistance you are looking for.

**Comment by potiuk** [comment]
Converting to discussion - until more information is provided. We can't deal with it as issue if we have no idea what happens.

You should see what argocd sync does with argocd support also you should look in k8s logs (not airflow logs) - but again this is argocd specific k8s deployment and likely you will find out how to do it with their docs and support.
